src/internal/components/token-list/select-toggle.tsx

```diff
@@ -1,29 +1,30 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
+
 import React, { useCallback } from 'react';
-import InternalIcon from '../icon/internal';
-import useFocusVisible from '../internal/hooks/focus-visible';
-import { NonCancelableEventHandler, fireNonCancelableEvent } from '../internal/events';
+import InternalIcon from '../../../icon/internal';
+import { fireNonCancelableEvent, NonCancelableEventHandler } from '../../events';
+import useFocusVisible from '../../hooks/focus-visible';
+import { I18nStrings } from './interfaces';
 
-import { TokenGroupProps } from './interfaces';
 import styles from './styles.css.js';
 export interface SelectToggleProps {
   controlId: string;
   allHidden: boolean;
   expanded: boolean;
   numberOfHiddenOptions: number;
   onClick?: NonCancelableEventHandler<null>;
-  i18nStrings?: Pick<TokenGroupProps.I18nStrings, 'limitShowFewer' | 'limitShowMore'>;
+  i18nStrings?: I18nStrings;
 }
 
-const SelectToggle = ({
+export function SelectToggle({
```

Comment:
minor: Probably out of scope, but the name of this component is confusing to me. Something like `DisclosureToggle`, `ListDisclosureToggle`, or `ItemLimitToggle` maybe?

Reply:
Thanks, I like the ItemLimitToggle 👍